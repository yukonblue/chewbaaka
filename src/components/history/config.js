/**
 * history/config.js
 * Chewbaaka
 *
 * Author   : Tomiko
 * Created  : Jul 07, 2020
 * Updated  : Aug 25, 2021
 */

export const config = {
  headMeta: {
    title: "History of the cheetah | Learn about cheetahs",
    description: "Learn about the history and evolution of the cheetah.",
    keywords: [
      "cheetah history",
      "cheetah evolution",
      "cheetah lineage",
      "cheetah population",
      "cheetah range",
    ]
  },
  pageProps: {
    coverImageNamePrefix: "History_Page_Cover_Image",
    title: "History",
    subtitle: "Learn about the evolution and migration of the species, its relationship with man throughout history, and its population and ranges.",
    pageMenuItems: [
      {
        title: "Evolution",
        tocImageFilename: "history_page_toc_menu_img_01-min.jpg"
      },
      {
        title: "Cheetah and Man",
        tocImageFilename: "history_page_toc_menu_img_02-min.jpg"
      },
      {
        title: "Range and Population",
        tocImageFilename: "history_page_toc_menu_img_03-min.jpg"
      }
    ],
    pageTailNavMenu: {
      prevPage: {
        label: "Home",
        href: "/"
      },
      nextPage: {
        label: "Biology",
        href: "/biology/"
      }
    }
  },
  contentPageIntro: {
    title: "History of the Cheetah",
    content: "The evolution and history of the cheetah are just as remarkable and " +
          "interesting as the species itself. Its evolution goes back to nearly " +
          "7 million years ago in time, and spanned across almost all the continents on the planet. " +
          "Its history with human interactions goes as far back as to 32000 BCE, and " +
          "the cheetah has been a symbol of significant meaning in different civilizations and " +
          "cultures across thousands of years of human history. Even up to this day, " +
          "the cheetah continues to be considered as a spiritual animal that manifests " +
          "as a symbol of a multitude of positive characteristics, such as adaptability, " +
          "persistence, and focus.",
    image: {
      filenamePrefix: "History_Page_Intro_Section_Background",
      aspectRatio: {
        width: 861,
        height: 420
      }
    }
  },
  contentPageSections: {
    section_Evolution: {
      intro: {
        title: "Evolution",
        content: "Although many would think the cheetah had originated on the continent of Africa, " +
        "the oldest fossils place  cheetahs in North America in what is now Texas, Nevada and Wyoming. " +
        "Cheetahs were common throughout Asia, Africa, Europe and North America until the end of the last " +
        "Ice Age, about 10,000 years ago, when massive climatic changes caused large numbers of mammals to disappear."
      },
      subsections: {
        subsection_EvolutionOfCats: {
          title: "Evolution of Cats"
        },
        subsection_CheetahEvolution: {
          title: "Cheetah Evolution"
        }
      },
    },
    section_CheetahAndMan: {
      intro: {
        title: "Cheetah and Man",
        content: "The history of cheetah-human relationship goes back in millennia, " +
        "and archeological artifacts uncovered throughout the world suggest that " +
        "human interactions with the species goes hand in hand with its migration and evolution. " +
        "The relationship between cheetahs and us have been both profound and complex, " +
        "as the species have been revered, utilized, displayed, and exploited throughout human history."
      },
      subsections: {
        subsection_RelationshipsWithMan: {
          title: "Relationships with Man",
          contents: {
            paragraph_Cheetah_And_Man_01:
              'Traditional African healers and witchdoctors used cheetah foot bones in spiritualistic rituals to symbolize fleet-footedness and speed. ' +
              'They used bones from a wide variety of animals and techniques varied from tribe to tribe. Objects ' +
              'represented a person, a thing or mood from the past, present or future, and are known as Divinatory sets.',
            paragraph_Cheetah_And_Man_02:
              'During the time of the Egyptian pharaohs, the cheetah was considered a goddess named ' +
              '"Mafdet". Pharaohs kept cheetahs as close companions, ' +
              "symbolic of Mafdet's protection of the royal throne.",
            paragraph_Cheetah_And_Man_03:
              'The San of Southern Africa ate cheetah meat as a symbol for speed, but it was not a main food in their diet.',
            paragraph_Cheetah_And_Man_04:
              'Kings wore cheetah skins for dignity. Trade in cheetah skin only started after European explorers began ' +
              'requesting them.',
          }
        },
        subsection_CheetahsInSports: {
          title: "Cheetahs In Sports",
          contents: {
            paragraph_The_Hunting_Leopard_01:
              'In c1700 BC, the Egyptians were the first to tame cheetahs. They admired the cheetah for its speed, ' +
              'hunting ability and beauty. They honored cheetahs as symbols of royalty and prestige.',

            paragraph_The_Hunting_Leopard_02:
              'The swiftest animal on earth became a cherished hunting companion of Pharaohs and royalty ' +
              'throughout Europe, Asia and India. Until the early 1900s, ownership of cheetahs was as important ' +
              'to these nobles as their love for gold. Cheetahs hunt by sight so they excelled in the sport ' +
              'known as "coursing". Hunts organized by royalty and nobles were for the challenge of sport, ' +
              'not for food. Hunts represented power and prestige.',

            paragraph_The_Hunting_Leopard_03:
              'By the 1500s the popularity of the cheetah as a hunting companion rivaled that of the dog. ' +
              'Cheetahs, the most easily tamed of the big cats, were caught, tamed and trained. Adults were ' +
              'used because cubs had not learned how to hunt. Tamed cheetahs formed a strong bond with their keepers.',

            paragraph_The_Hunting_Leopard_04:
              'Each cheetah rode to the hunt by horseback or on a cart. Its eyes were coved with a hood and uncovered ' +
              'when prey were sighted. The cat was released to chase down the prey then rewarded with meat fed from a ' +
              'wooden spoon.',

            paragraph_The_Hunting_Leopard_05:
              'Although cherished, pampered cheetahs were loved to near extinction and taken from the wild in great numbers. ' +
              'By the early 1900s, India and Iran were importing African cheetahs for the sport of "coursing", as their own ' +
              'wild population became too small.'
          }
        },
        subsection_CheetahsInArt: {
          title: "Cheetahs In Art",
          contents: {
            paragraph_Cheetah_In_Art_01:
              'Sumerians (Iraqi ancestors) were the first to use cheetahs in art.',

            paragraph_Cheetah_In_Art_02:
              'The cheetah and leopard have often been mistaken for each other. ' +
              'Cheetahs in early art were frequently called "panther" ' +
              'or "hunting leopard". Early artists called the true leopard "pardus" ' +
              'and described it as a cross between a cheetah (panther), and the lion (leo).',

            paragraph_Cheetah_In_Art_03:
              'Middle Age and Renaissance artists began drawing cheetahs and other ' +
              'animals more life-like. During this time of scientific exploration, ' +
              'detailed descriptions of animals provided incentive for reality in art. ' +
              'Artists included cheetahs more often in their paintings as the trade in ' +
              'live animals increased.',

            paragraph_Cheetah_In_Art_04:
              'Artists always pictured cheetahs as animals of speed and royalty.'
          }
        }
      }
    },
    section_PopulationAndRange: {
      intro: {
        title: "Range and Population",
        content: "A combination of factors such as game hunting, retaliatory killing, " +
                "habitat encroachment have caused the sharp decline of the global cheetah " +
                "population over the last century, and driven the species out of much of their " +
                "historical range. One of the last strongholds of the species today remain in " +
                "southern Africa, spanning over Namibia and Botswana, which together " +
                "holds two thirds of the species' population."
      },
      subsections: {
        subsection_Namibia: {
          title: "Namibia - Cheetah Capital of the World",
          contents: {
            Cheetah_Capital_Of_The_World: {
              title: "Namibia",
              subtitle: "Cheetah Capital of the World",
              content: {
                paragraph_Cheetah_Capital_Of_The_World_01:
                  "Namibia has the world's largest cheetah population. " +
                  "Approximately 3000 cheetahs share the land with humans, " +
                  "livestock, and wildlife.",

                paragraph_Cheetah_Capital_Of_The_World_02:
                  "Today, the status of the Namibian cheetah is stabilizing. " +
                  "During the 1980s, the population of Namibian cheetahs " +
                  "declined by half. In the 10-year period, nearly " +
                  "7,000 cheetahs were removed from the wild.",

                paragraph_Cheetah_Capital_Of_The_World_03:
                  "In most countries where cheetahs live, their numbers " +
                  "have been reduced to critical levels. The cheetah's " +
                  "survival worldwide is in human hands."
              }
            },
            Namibia_Home_For_The_Future: {
              title: "Namibia",
              subtitle: "Hope for the Future",
              content: {
                paragraph_01:
                  "The greatest hope for the cheetah's survival lies in the " +
                  'pristine countryside of Namibia. Nearly 1000 Namibian farmers ' +
                  "control the fate of the country's cheetahs and the land they live."
              }
            }
          }
        },
        subsection_RoadToExtinction: {
          title: "The Road to Extinction",
          contents: {
            The_Road_To_Extinction: {
              title: "The Road to Extinction?",
              subtitle: "1990",
              content: {
                paragraph_The_Road_To_Extinction_01:
                  "World development, industrialization, automobiles, and airplanes...",

                paragraph_The_Road_To_Extinction_02:
                  "Through the 1900s, man's inventions seemed limitless, yet there is " +
                  "nothing man-made that rivals the speed and efficiency of the cheetah.",

                paragraph_The_Road_To_Extinction_03:
                  "Scientists classified the cheetah as Acinonyx jubatus, yet nobles still " +
                  'referred to it as the "hunting leopard". By the end of the 1800s, cheetahs ' +
                  'were a rarity in Asia Minor and Arabia because of their use in the sport ' +
                  'of "coursing".',

                paragraph_The_Road_To_Extinction_04:
                  'Although it appeared that the cheetahs had a large range, their numbers ' +
                  'within that range existed in small pockets. Cheetahs, farmers and their livestock ' +
                  'all preferred open grasslands for their habitat. Increasing agricultural development ' +
                  'and new settlements played havoc with remaining cheetah populations.',
              }
            },
            Where_Did_The_Cheetah_Go: {
              title: "Where Did the Cheetah Go?",
              subtitle: "1975",
              content: {
                paragraph_Where_Did_The_Cheetah_Go_01:
                  'As human population grew, the amount of land devoted to livestock farming ' +
                  'steadily increased. Livestock filled the open land where cheetahs roamed. ' +
                  'Natural prey became scarce. Farmers kill other large predators. Although ' +
                  'game reserves protected them, cheetahs could not compete against hyenas ' +
                  'and lions. Farmlands offered cheetahs a safe haven, but they sometimes killed livestock.',

                paragraph_Where_Did_The_Cheetah_Go_02:
                  'Farmers saw cheetahs more frequently and thought their numbers had increased. ' +
                  'Cheetahs took the blame for most predator-related livestock losses. ' +
                  'Farmers killed cheetahs by the thousands as pests or to sell their skins ' +
                  'to the fur trade.',

                paragraph_Where_Did_The_Cheetah_Go_03:
                  'By 1975, researches realized that the cheetah was in trouble. CITES ' +
                  '(Convention on International Trade of Endangered Species) placed the cheetah ' +
                  'on Appendix I, making international trade in live cheetah or cheetah products illegal. ' +
                  'Local laws supported CITES in many countries where cheetahs still lived. ' +
                  'Researchers began looking for ways to encourage the growth of cheetah populations ' +
                  'through land management practices.'
              }
            }
          }
        }
      }
    }
  }
};
