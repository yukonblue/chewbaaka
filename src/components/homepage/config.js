/**
 * homepage/config.js
 * Chewbaaka
 *
 * Author   : Tomiko
 * Created  : Jul 28, 2020
 * Updated  : Aug 17, 2021
 */

export const config = {
  components: {
    landingPageGateway: {
      constants: {
        landingPageGatewayTitle: "Learn about cheetahs",
        landingPageGatewaySubtitle: "Explore the Cheetah Museum at Cheetah Conservation Fund"
      },
    }
  }
}
