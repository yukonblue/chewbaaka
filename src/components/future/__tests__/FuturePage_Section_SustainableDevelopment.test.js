/**
 * FuturePage_Section_SustainableDevelopment.test.js
 * Chewbaaka
 *
 * Author   : Tomiko
 * Created  : Jul 22, 2020
 * Updated  : Aug 21, 2021
 */

import React from 'react'

import { render } from '@testing-library/react'

import { config } from '../config'

import renderer from 'react-test-renderer'

import FuturePageSectionSustainableDevelopment from '../FuturePage_Section_SustainableDevelopment'

test('renders FuturePageSectionSustainableDevelopment component', () => {
  render(
    <FuturePageSectionSustainableDevelopment
      config={config}
    />
  );
});

test('FuturePageSectionSustainableDevelopment component snapshot', () => {
  const tree = renderer
    .create(
      <FuturePageSectionSustainableDevelopment
        config={config}
      />
    ).toJSON();
  expect(tree).toMatchSnapshot();
});
