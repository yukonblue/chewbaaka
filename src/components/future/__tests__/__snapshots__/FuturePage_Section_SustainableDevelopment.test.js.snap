// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FuturePageSectionSustainableDevelopment component snapshot 1`] = `
<article
  className="ContentPageSectionTemplateOuterContainer"
>
  <h2
    className="ContentPageSectionHeadTitle"
    id="sustainable-development"
  >
    Sustainable Development
  </h2>
  <p
    className="ContentPageHeadAndSectionIntroText"
  >
    Sustainable development is key to ensure long term utilizations of natural resources without compromising the integrity of the ecosystem and the sharing of resources by other species. Part of sustainable development involves taking part of the revenue generated from any human development and contribute that to the nurturing and protection of the entire ecosystem. Any human development can be turned to be sustainable by employing the best practices of land, livestock, and wildlife management techniques that have proven to be effective and successful.
  </p>
  <section
    className="ContentPageSubsectionTemplateOuterContainer"
  >
    <h3
      className="ContentPageSubsectionTitle"
    >
      Mission Possible
    </h3>
    <div>
      <div
        className="ContentPageSubsectionPartContainer"
      >
        <p
          className="ContentPageRegularTextSize ContentPageRegularTextLineHeight ContentPageContentParagraphText"
        >
          In Africa, property that is owned and managed by farmers can maintain viable populations of animals and natural habitats. It is the careful management of these habitats that holds the key to the future survival of plant and animal species such as the cheetah.
        </p>
        <p
          className="ContentPageRegularTextSize ContentPageRegularTextLineHeight ContentPageContentParagraphText"
        >
          In order to ensure a successful future we need to be responsible custodians of nature. The way land and animals are managed determines the future of all ecosystems.
        </p>
      </div>
      <div
        className="ContentPageSubsectionPartContainer"
      >
        <div
          className="ContentPageSubsectionThreeColumnContentTemplateOuterContainer"
        >
          <div
            className="ContentPageSubsectionThreeColumnContentTemplateInnerContainer"
          >
            <div
              className="ContentPageSubsectionThreeColumnContentTemplateColumn ContentPageSubsectionThreeColumnContentTemplateLeftColumnContainer"
            >
              <h4
                className="ContentPageSubsectionSubtitle"
              >
                Land Management
              </h4>
              <div>
                <p
                  className="ContentPageRegularTextSize ContentPageRegularTextLineHeight ContentPageContentParagraphText"
                >
                  The productivity of the land depends on water and soil. Preventing erosion increases the potential for the land to produce food and allows water to penetrate the soil, thus carrying important nutrients to the roots of the plants. We must learn to live within the limits of the scarcity of water in our dry country.
                </p>
                <p
                  className="ContentPageRegularTextSize ContentPageRegularTextLineHeight ContentPageContentParagraphText"
                >
                  The use of alternative fuels or efficient wood burning stoves and ovens reduces deforestation. Establishing timber plantations and using alternative building materials saves natural forests.
                </p>
              </div>
            </div>
            <div
              className="ContentPageSubsectionThreeColumnContentTemplateColumn ContentPageSubsectionThreeColumnContentTemplateMiddleColumnContainer"
              style={
                Object {
                  "backgroundColor": "rgb(undefined)",
                }
              }
            >
              <h4
                className="ContentPageSubsectionSubtitle"
              >
                Livestock Management
              </h4>
              <div>
                <div>
                  <p
                    className="ContentPageRegularTextSize ContentPageRegularTextLineHeight ContentPageContentParagraphText"
                  >
                    Many farmers use rotational grazing where farmlands are divided into camps. Practicing rotational and seasonal grazing allows used areas time to recover.
                  </p>
                  <p
                    className="ContentPageRegularTextSize ContentPageRegularTextLineHeight ContentPageContentParagraphText"
                  >
                    Recovery of damaged land occurs faster if it is protected from the impact of overgrazing. Proper herd size is necessary to prevent overgrazing, trampling of the land and reducing predator problems. A balance of grazing and browsing animals reduces the pressure on one vegetation type.
                  </p>
                  <img
                    alt="Sustainable development"
                    className="FullWidth"
                    height={640}
                    src="Sustainable_Development-min.png"
                    width={640}
                  />
                </div>
              </div>
            </div>
            <div
              className="ContentPageSubsectionThreeColumnContentTemplateColumn ContentPageSubsectionThreeColumnContentTemplateRightColumnContainer"
            >
              <h4
                className="ContentPageSubsectionSubtitle"
              >
                Wildlife Management
              </h4>
              <div>
                <p
                  className="ContentPageRegularTextSize ContentPageRegularTextLineHeight ContentPageContentParagraphText"
                >
                  Allowing wild animals to migrate naturally through farm areas promotes the balance of browsers and grazers, and allows predators their food at wild game. This reduces the temptation to take domestic stock.
                </p>
                <p
                  className="ContentPageRegularTextSize ContentPageRegularTextLineHeight ContentPageContentParagraphText"
                >
                  Ethical hunting removes older animals while predators prey on the young, diseased, and genetically weak. Together they keep wild herds healthy and in check.
                </p>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
  </section>
  <section
    className="ContentPageSubsectionTemplateOuterContainer"
  >
    <h3
      className="ContentPageSubsectionTitle"
    >
      Future Farmers of Africa
    </h3>
    <div>
      <div
        className="ContentPageSubsectionPartContainer"
      >
        <div
          className="ContentPageSideFloatFluidContainerOuterContainer"
        >
          <div
            className="ContentPageSideFloatFluidContainerFloatPartContainer BottomMargin20px FloatRight"
          >
            <div
              style={
                Object {
                  "width": 240,
                }
              }
            >
              <div
                className="ui inverted segment StatsLabel"
              >
                <div
                  className="ui orange huge inverted statistic"
                >
                  <div
                    className="value"
                  >
                    90%
                  </div>
                  <div
                    className="label"
                  >
                    of the cheetah population in Namibia lives on farmlands alongside 80% of the country’s wildlife species
                  </div>
                </div>
              </div>
            </div>
          </div>
          <div
            className="ContentPageSideFloatFluidContainerFixedPartContainer"
          >
            <p
              className="ContentPageRegularTextSize ContentPageRegularTextLineHeight ContentPageContentParagraphText"
            >
              In Namibia, 90% of the cheetah population lives on farmlands alongside 80% of the country’s wildlife species. Cheetahs often come into contact with the livestocks and game farming communities, leading to conflict. To mitigate this conflict, Cheetah Conservation Fund (CCF) developed an integrated livestock, wildlife, and rangeland training program called Future Farmers of Africa (FFA).
            </p>
          </div>
        </div>
      </div>
      <div
        className="ContentPageSubsectionPartContainer"
      >
        <div
          className="ContentPageSideFloatFluidContainerOuterContainer"
        >
          <div
            className="ContentPageSideFloatFluidContainerFloatPartContainer BottomMargin20px FloatLeft"
          >
            <div
              className="ImageViewOuterContainer"
              style={
                Object {
                  "maxWidth": 742,
                }
              }
            >
              <div
                className="ImageViewInnerContainer ImageViewFamilyContainerBorderStyles ImageViewFamilyContainerShadow ImageViewFamilyContainerBackgroundStyles"
              >
                <div
                  className="ImageViewImgContainer"
                >
                  <img
                    alt="Farmers participate in FFA training courses."
                    className="ImageViewFamilyImgLayoutStyles ImageViewFamilyImgBorderStyles"
                    height={370}
                    src="CCF_FFA_01-min.jpg"
                    width={742}
                  />
                </div>
                <div
                  className="ImageViewFamilyCaptionContainerStyles"
                >
                  <div
                    className="CaptionContainer"
                    data-testid="CaptionComponentTestId"
                    style={
                      Object {
                        "maxWidth": 742,
                      }
                    }
                  >
                    <div
                      className="IconContainer"
                    >
                      <i
                        aria-hidden="true"
                        className="image icon"
                        onClick={[Function]}
                      />
                    </div>
                    <p
                      className="ContentPageCaptionTextSize CaptionText"
                    >
                      Farmers participate in FFA training courses. (Image credit: Cheetah Conservation Fund)
                    </p>
                  </div>
                </div>
              </div>
            </div>
          </div>
          <div
            className="ContentPageSideFloatFluidContainerFixedPartContainer"
          >
            <div>
              <p
                className="ContentPageRegularTextSize ContentPageRegularTextLineHeight ContentPageContentParagraphText"
              >
                From 2006-2017, CCF has certified more than 10,000 Namibian men and women in FFA, which teaches conservation strategies for sustainable land use while accommodating the coexistence of predators alongside livestock and wildlife.
              </p>
              <p
                className="ContentPageRegularTextSize ContentPageRegularTextLineHeight ContentPageContentParagraphText"
              >
                Topics covered in the FFA courses include livestock health and veterinary care, livestock husbandry, livestock valuation, predator spoor (track) identification, differentiating predator kill techniques, best practices to reduce livestock losses, and best methods for non-lethal predator control.
              </p>
              <p
                className="ContentPageRegularTextSize ContentPageRegularTextLineHeight ContentPageContentParagraphText"
              >
                Through the training course, CCF supports farmers by teaching them practical skills in sustainable livestock farming, as well as how to develop supplemental income streams. Farmers then use their skills to generate higher economic productivity. These courses enhance the livelihood development of farmers while promoting predator-friendly farming practices.
              </p>
            </div>
          </div>
        </div>
      </div>
      <div
        className="ContentPageSubsectionPartContainer"
      >
        <p
          className="ContentPageRegularTextSize ContentPageRegularTextLineHeight ContentPageContentParagraphText"
        >
          CCF’s Integrated Livestock and Wildlife Model Farm houses cattle, goats, sheep and livestock guarding dogs, as well as Namibian wildlife. The Model Farm serves as a training facility for FFA, demonstrating how sustainable livestock farming practices allow for the management of their livestock and better livestock health without having to eliminate cheetahs and other predators.
        </p>
        <p
          className="ContentPageRegularTextSize ContentPageRegularTextLineHeight ContentPageContentParagraphText"
        >
          CCF reaches out to local farming communities, visiting farms and joining annual farmer’s association meetings to give vital advice on improvements that farmers can make to their rangeland, livestock and wildlife management.
        </p>
        <div
          className="VerticalCushionPadding"
        >
          <div>
            <a
              href="https://cheetah.org/canada/about-us/what-we-support/future-farmers-of-africa/"
            >
              <button
                className="ui blue icon button"
                onClick={[Function]}
              >
                Read on CCF's blog on Future Farmers of Africa
                <i
                  aria-hidden="true"
                  className="file alternate outline icon"
                  onClick={[Function]}
                />
              </button>
            </a>
          </div>
        </div>
      </div>
    </div>
  </section>
  <section
    className="ContentPageSubsectionTemplateOuterContainer"
  >
    <h3
      className="ContentPageSubsectionTitle"
    >
      Livestock Guarding Dogs
    </h3>
    <div>
      <div
        className="ContentPageSubsectionPartContainer"
      >
        <h4
          className="ContentPageSubsectionSubtitle"
        >
          CCF’s Livestock Guarding Dog History
        </h4>
        <p
          className="ContentPageRegularTextSize ContentPageRegularTextLineHeight ContentPageContentParagraphText"
        >
          In 1994, CCF created a solution to non-lethal predator control - Livestock Guarding Dogs (LGD). After extensive research, CCF felt that the Anatolian Shepherd and Kangal dogs best met the livestock managers’ needs in Namibia. In cooperation with Dr. Ray Coppinger from the Livestock Guarding Dog Association in Hampshire College (Massachusetts, U.S.), the first 10 dogs were donated to Namibia by Birinci Anatolians.
        </p>
        <p
          className="ContentPageRegularTextSize ContentPageRegularTextLineHeight ContentPageContentParagraphText"
        >
          Since then, hundreds of farmers have received Anatolian Shepherds and Kangal dogs to ward off predators, such as cheetahs, jackals and leopards. These dogs do not herd the goats, but protect the flock by placing themselves between the herd and the predator and barking loudly.
        </p>
        <p
          className="ContentPageRegularTextSize ContentPageRegularTextLineHeight ContentPageContentParagraphText"
        >
          CCF’s success is due to the commitment of CCF staff members and the participating farmers. CCF staff regularly visit the farms, educating and advising the farmers and herders for the dogs to be as effective as possible.
        </p>
        <div
          className="VerticalCushionPadding"
        >
          <div
            className="Centered FullWidth"
            style={
              Object {
                "width": 1200,
              }
            }
          >
            <img
              alt="Livestock Guarding Dog"
              className="FullWidth DisplayBlock RetainAspectRatio"
              height={809}
              src="CCF_LGD_L-min.jpg"
              width={1200}
            />
          </div>
        </div>
      </div>
      <div
        className="ContentPageSubsectionPartContainer"
      >
        <h4
          className="ContentPageSubsectionSubtitle"
        >
          Raising LGD’s
        </h4>
        <div
          className="ContentPageSideFloatFluidContainerOuterContainer"
        >
          <div
            className="ContentPageSideFloatFluidContainerFloatPartContainer BottomMargin20px FloatRight"
          >
            <div
              className="ImageViewOuterContainer"
              style={
                Object {
                  "maxWidth": 720,
                }
              }
            >
              <div
                className="ImageViewInnerContainer ImageViewFamilyContainerBorderStyles ImageViewFamilyContainerShadow ImageViewFamilyContainerBackgroundStyles"
              >
                <div
                  className="ImageViewImgContainer"
                >
                  <img
                    alt="Anatolian Shepherd puppies live alongside with the herd."
                    className="ImageViewFamilyImgLayoutStyles ImageViewFamilyImgBorderStyles"
                    height={480}
                    src="CCF_anatolian_shepherd_puppies-min.jpg"
                    width={720}
                  />
                </div>
                <div
                  className="ImageViewFamilyCaptionContainerStyles"
                >
                  <div
                    className="CaptionContainer"
                    data-testid="CaptionComponentTestId"
                    style={
                      Object {
                        "maxWidth": 720,
                      }
                    }
                  >
                    <div
                      className="IconContainer"
                    >
                      <i
                        aria-hidden="true"
                        className="image icon"
                        onClick={[Function]}
                      />
                    </div>
                    <p
                      className="ContentPageCaptionTextSize CaptionText"
                    >
                      Anatolian Shepherd puppies live alongside with the herd. (Image credit: Cheetah Conservation Fund)
                    </p>
                  </div>
                </div>
              </div>
            </div>
          </div>
          <div
            className="ContentPageSideFloatFluidContainerFixedPartContainer"
          >
            <p
              className="ContentPageRegularTextSize ContentPageRegularTextLineHeight ContentPageContentParagraphText"
            >
              LGD’s are placed with their herds around nine weeks of age, as social bonding between the stock and the dogs deepens upon imprinting before 16 weeks of age. The dogs live with their livestock family instead of a human family, so the puppies bond with the herd; it becomes the dog’s pack rather than the dog’s potential prey. At CCF puppies are born in a whelping pen attached to the main goat yard.
            </p>
          </div>
        </div>
      </div>
      <div
        className="ContentPageSubsectionPartContainer"
      >
        <div
          className="Centered FullWidth"
          style={
            Object {
              "width": 1000,
            }
          }
        >
          <div
            className="ImageViewOuterContainer"
            style={
              Object {
                "maxWidth": 1000,
              }
            }
          >
            <div
              className="ImageViewInnerContainer ImageViewFamilyContainerBorderStyles ImageViewFamilyContainerShadow ImageViewFamilyContainerBackgroundStyles"
            >
              <div
                className="ImageViewImgContainer"
              >
                <img
                  alt="Established in 1994, 2019 marked the 25th anniversary of CCF's Livestock Guarding Dog program."
                  className="ImageViewFamilyImgLayoutStyles ImageViewFamilyImgBorderStyles"
                  height={299}
                  src="CCF_Year_of_the_LGD-min.jpg"
                  width={1000}
                />
              </div>
              <div
                className="ImageViewFamilyCaptionContainerStyles"
              >
                <div
                  className="CaptionContainer"
                  data-testid="CaptionComponentTestId"
                  style={
                    Object {
                      "maxWidth": 1000,
                    }
                  }
                >
                  <div
                    className="IconContainer"
                  >
                    <i
                      aria-hidden="true"
                      className="image icon"
                      onClick={[Function]}
                    />
                  </div>
                  <p
                    className="ContentPageCaptionTextSize CaptionText"
                  >
                    Established in 1994, 2019 marked the 25th anniversary of CCF's Livestock Guarding Dog program. (Image credit: Cheetah Conservation Fund)
                  </p>
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
      <div
        className="ContentPageSubsectionPartContainer"
      >
        <h4
          className="ContentPageSubsectionSubtitle"
        >
          Dogs Save Cheetahs
        </h4>
        <div
          className="ContentPageSideFloatFluidContainerOuterContainer"
        >
          <div
            className="ContentPageSideFloatFluidContainerFloatPartContainer BottomMargin20px FloatRight"
          >
            <img
              alt="Cheetah and LGD"
              className="FullWidth DisplayBlock RetainAspectRatio"
              height={576}
              src="Cheetah_and_LGD-min.jpg"
              width={720}
            />
          </div>
          <div
            className="ContentPageSideFloatFluidContainerFixedPartContainer"
          >
            <p
              className="ContentPageRegularTextSize ContentPageRegularTextLineHeight ContentPageContentParagraphText"
            >
              Livestock Guarding Dogs have been used for thousands of years to protect cattle, sheep and goats from predator attacks. There are over 20 breeds of guarding dogs. The Anatolian Shepherd, a Turkish breed, was selected as the best candidate for use with smallstock in Namibia as they are able to work in vast open spaces without direct guidance. These dogs look similar to the flock, with large rounded heads, floppy ears and short fur. They have a good sense of hearing and smelling, a calm temperament and a very loud bark.
            </p>
            <p
              className="ContentPageRegularTextSize ContentPageRegularTextLineHeight ContentPageContentParagraphText"
            >
              Anatolians do not herd livestock, they guard them. They are attentive, protective, trustworthy, and aggressive towards predator threats.
            </p>
            <p
              className="ContentPageRegularTextSize ContentPageRegularTextLineHeight ContentPageContentParagraphText"
            >
              Puppies are placed with the stock at eight weeks of age to form a strong bond with the herd. It is important that bonding occurs with the herd and not with humans or other dogs.
            </p>
            <p
              className="ContentPageRegularTextSize ContentPageRegularTextLineHeight ContentPageContentParagraphText"
            >
              Dog owners take the responsibility for the health care of their dogs. This includes veterinary check-ups, vaccinations, and observations for signs of illness or injuries.
            </p>
            <p
              className="ContentPageRegularTextSize ContentPageRegularTextLineHeight ContentPageContentParagraphText"
            >
              An appropriate diet is necessary for the Anatolian Shepherd. Enough food allows proper growth and a healthy dog. They should never be given raw meat as this could produce a predatory response. They need water during the day when they are out with the herd.
            </p>
          </div>
        </div>
        <div
          className="VerticalCushionPaddingTopLarge"
        >
          <img
            alt="Livestock Guarding Dogs"
            className="FullWidth DisplayBlock RetainAspectRatio Centered"
            height={960}
            src="LGD_960x960_L-min.png"
            width={960}
          />
        </div>
      </div>
    </div>
  </section>
</article>
`;
